app/models/condition.rb

```diff
@@ -1,3 +1,37 @@
+# class:          Condition
+#
+# Responsibility: Provides the information needed so that a specific
+#                 class can be instantiated and :process_condition
+#                 can run on those instances.
+#
+#                 This class is  _descriptive._  (= the configration pattern)
+#
+#                 Ex:  A nightly rake/rails task will go thru all of the
+#                 Condition record in the db and, using the information in each,
+#                 instantiate the class in _class_name_.
+#                 The task will send each instance the :process_condition method,
+#                 passing it the name, timing, and config data.
+#
+#                 Each instance will then do 'whatever needs to be done' when
+#                 it is sent :process_condition with the information.
+#                 Ex: this will send out emails to all users that are past due
+#                 with their Membership fee, etc.
+#
+#
+# Attributes: These are all essentially *class variables* as they apply
+#   to every instance of a class.
+#
+#   class_name - the name of the class to instantiate
+#   name - the name of the method to call for the condition TODO - is this really needed?
```

Comment:
I think we should drop it for now and then can add it later if it's called for.  
I don't believe we should create a class whose responsibility is to run more than 1 condition.  That means that the class has more than 1 responsibility.  
Hence I don't think it will be needed.

Reply:
OK, lets drop it.  Add to this PR or separately?

Reply:
done in this PR.